package.json

```diff
@@ -41,7 +41,8 @@
     "dev:bp": "cd packages/boilerplate/theme && yarn dev",
     "dev:ct": "cd packages/commercetools/theme && yarn dev",
     "link-packages": "lerna link --force-local",
-    "lint": "eslint . --ext .js,.ts,.vue"
+    "lint": "eslint . --ext .js,.ts,.vue",
+    "test-cli": "cd packages/core/cli/src/scripts && ts-node createProject.ts"
```

Comment:
**change:** Imho it's not needed here. If you want to test it - write a test ;)

Reply:
that's not needed,  please use `bin` section in the `package.json` of CLI package, then once you have installed package you can just use `yarn [your cli name]` () and that's it!

```
"bin": {
  "vsf": "ts-nodd cli.ts"
}
```

Reply:
btw. with using this we probably will be able to use this package with npx

Reply:
@andrzejewsky yup thats the next step ;)

Reply:
also you can call @Fifciu "Jędrek" or "Jendre" (more international).

Reply:
Call me "Fifciuu" guys. Double-U is important. Nonetheless, Jendre sounds funny :D